### PR TITLE
Optimizations

### PR DESCRIPTION
**Pull request by thevindu-w**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -8,7 +7,8 @@ logs
 .vscode
 .idea
 .dockerignore
+Dockerfile
```
Comment (Ishad-M-I-M) [inline .dockerignore:10]: Why Dockerfile ignored

File: tests/unit/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+FROM miyurud/jasminegraph:20230909T110050
```
Comment (Ishad-M-I-M) [inline tests/unit/Dockerfile:1]: For future unit tests, they need to be run in a k8s pod in order to establish connection with the kubernetes API server.
This kind of thing
https://github.com/FYP-Auto-Scale-JasmineGraph/jasminegraph/pull/21/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R30

File: .dockerignore (hunk 1 above)
Comment (thevindu-w) [reply .dockerignore:10]: Don't need it inside container

File: tests/unit/util/utils_test.cpp [hunk 3]
```diff
@@ -1,8 +1,25 @@
...
+
+    std::sort(flist.begin(), flist.end());
+    ASSERT_EQ(flist.at(0), "CMakeCache.txt");
+    ASSERT_EQ(flist.at(1), "CTestTestfile.cmake");
+    ASSERT_EQ(flist.at(2), "JasmineGraph");
+    ASSERT_EQ(flist.at(3), "Makefile");
+    ASSERT_EQ(flist.at(4), "cmake_install.cmake");
+    ASSERT_EQ(flist.at(5), "libJasmineGraphLib.a");
+
+    ASSERT_EQ(Utils::getJasmineGraphProperty("org.jasminegraph.server.host"), "localhost");
```
Comment (Ishad-M-I-M) [inline tests/unit/util/utils_test.cpp]: This is not related to this test case
```suggestion
```

File: src/util/Utils.cpp [hunk 4]
```diff
@@ -227,7 +218,6 @@ bool Utils::is_number(const std::string &compareString) {
  * @return
  */
 std::string Utils::getFileName(std::string filePath) {
-    // FIXME: Can file path separator be '\' on Linux?
```
Comment (Ishad-M-I-M) [inline src/util/Utils.cpp:230]: Is this resolved?.

Comment (thevindu-w) [reply src/util/Utils.cpp:230]: File path separator is / in Linux.

File: tests/unit/Dockerfile (hunk 2 above)
Comment (Ishad-M-I-M) [reply tests/unit/Dockerfile:1]: Done

**Comment by Ishad-M-I-M** [comment]
I have renamed the `utils_test.cpp` file to `Utils_test.cpp`. I think better to rename it here as well unless It'll create a conflict
